src/main/java/com/hashmapinc/tempus/WitsmlObjects/v1411/ObjWellbore.java

```diff
@@ -400,7 +393,7 @@ public void setShape(WellboreShape value) {
      *     {@link XMLGregorianCalendar }
      *
      */
-    public XMLGregorianCalendar getDTimKickoff() {
+    public String getDTimKickoff() throws DatatypeConfigurationException {
```

Comment:
@cherrera2001 	why does this throw DatatypeConfigurationException?

Reply:
because of the verification step to actually make sure it is a valid xmldatetime. the DataTypeFactory.newInstance() throws that...it could be caught in the getter though.